spec/rails_helper.rb

```diff
@@ -6,6 +6,7 @@
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
 require 'json_expressions/rspec'
+require 'webmock/rspec'
```

Comment:
**[nit pick/not a blocker]** I believe this is already required in the [spec_helper](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/29dce121c8c27f238e004951cc331ade8f4533d4/spec/spec_helper.rb#L17), so shouldn't really be necessary to require it here.